bundles/org.openhab.binding.comfoair/src/main/java/org/openhab/binding/comfoair/internal/ComfoAirHandler.java

```diff
@@ -0,0 +1,347 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.comfoair.internal;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+
+import org.apache.commons.lang.StringUtils;
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.smarthome.core.library.types.DecimalType;
+import org.eclipse.smarthome.core.library.types.OnOffType;
+import org.eclipse.smarthome.core.library.types.QuantityType;
+import org.eclipse.smarthome.core.library.unit.SIUnits;
+import org.eclipse.smarthome.core.library.unit.SmartHomeUnits;
+import org.eclipse.smarthome.core.thing.Channel;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingStatusDetail;
+import org.eclipse.smarthome.core.thing.binding.BaseThingHandler;
+import org.eclipse.smarthome.core.types.Command;
+import org.eclipse.smarthome.core.types.RefreshType;
+import org.eclipse.smarthome.core.types.State;
+import org.eclipse.smarthome.core.types.UnDefType;
+import org.eclipse.smarthome.io.transport.serial.SerialPortManager;
+import org.openhab.binding.comfoair.internal.datatypes.ComfoAirDataType;
+import org.openhab.binding.comfoair.internal.datatypes.DataTypeBoolean;
+import org.openhab.binding.comfoair.internal.datatypes.DataTypeNumber;
+import org.openhab.binding.comfoair.internal.datatypes.DataTypeRPM;
+import org.openhab.binding.comfoair.internal.datatypes.DataTypeTemperature;
+import org.openhab.binding.comfoair.internal.datatypes.DataTypeVolt;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link ComfoAirHandler} is responsible for handling commands, which are
+ * sent to one of the channels.
+ *
+ * @author Hans Böhm - Initial contribution
+ */
+@NonNullByDefault
+public class ComfoAirHandler extends BaseThingHandler {
+    private static final int DEFAULT_REFRESH_INTERVAL = 60;
+
+    private final Logger logger = LoggerFactory.getLogger(ComfoAirHandler.class);
+    private @Nullable ScheduledFuture<?> poller;
+    private SerialPortManager serialPortManager;
```

Comment:
```suggestion
    private final SerialPortManager serialPortManager;
```
Also move this field above the non-final fields.